certified-connectors/Tribal - Maytas/README.md

```diff
@@ -0,0 +1,51 @@
+# Tribal Connector
+
+Streamline the day-to-day administration of training, reduce administration costs and improving efficiency, manage courses, monitor learner success rates.
```

Comment:
Hello @mszy-tribal

Kindly update readme.md file as per the template in the below url. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
done